JapaneseFilter/sections/specific.txt

```diff
@@ -907,7 +907,11 @@ news-channel.doorblog.jp##.headerBnr
 ||0matome.com/img/pr_k-jz.png
 ||fc2.com/distribution/preroll/
 keibana.com##.ad-g
-keibana.com##.l-header__ad
+iko-yo.net##.c-ad-information
+iko-yo.net##.c-ad--h250
+iko-yo.net##.c-ad--tieup
+iko-yo.net##.c-ad_container
+iko-yo.net,keibana.com##.l-header__ad
```

Comment:
Please also add `iko-yo.net##.c-ad--w468`, it's in the top page `https://iko-yo.net/`.

Reply:
ok.